data/json/items/gun/223.json

```diff
@@ -764,5 +764,79 @@
     "pocket_data": [
       { "pocket_type": "MAGAZINE_WELL", "item_restriction": [ "augmag_30rd", "augmag_10rd", "augmag_42rd", "augmag_100rd" ] }
     ]
+  },
+  {
+    "id": "zpapm90",
+    "copy-from": "rifle_semi",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "ZPAP M90" },
+    "description": "Civilian AK pattern rifle made by Zastava in 5.56 NATO.",
```

Comment:
```suggestion
    "name": { "str": "5.56x45mm AK" },
    "description": "A civilian AK pattern rifle, chambered in 5.56x45 NATO.",
    "variant_type": "gun",
    "variants": [
      {
        "id": "zpapm90",
        "name": { "str": "ZPAP M90" },
        "description": "A Serbian-made AK pattern rifle made by Zastava in 5.56x45mm NATO. To avoid import restrictions, Zastava USA has added a number of American-made parts.",
        "weight": 1
      }
    ],
```